projects/systelab-components/sass/modern/_variables.scss

```diff
@@ -12,6 +12,8 @@
   --slab_background_primary: white;
   --slab_foreground_primary: black;
 
+  --slab_numpad_icon: rgb(52, 58, 64);
```

Comment:
You should use a variable from the file _theme.config.ts_ so it changes with the theme. Otherwise it will be the same no matter what theme the application is using.